packages/composites/google-weather/README.md

```diff
@@ -35,7 +35,7 @@ env PROJECT_ID=gcp-pdp-weather-dev yarn start
 |    ✅     | `dateFrom` |         The date to query data from (inclusive) in ISO 8601 format         |                                   |             |
 |    ✅     |  `dateTo`  |          The date to query data to (inclusive) in ISO 8601 format          |                                   |             |
 |    ✅     |  `method`  |                  Which method to use to aggregate data in                  |    `AVG`, `SUM`, `MIN`, `MAX`     |             |
-|    ✅     |  `field`   |                      Which column to fetch data from                       | [Data available](#data-available) |             |
+|    ✅     |  `column`  |                      Which column to fetch data from                       | [Data available](#data-available) |             |
```

Comment:
Has this been changed in the code as well?

Reply:
The code always used the param name `column`, so only a change in the README was necessary